app/src/main/java/nl/nfi/cellscanner/UserDataUploadWorker.java

```diff
@@ -86,11 +90,11 @@ public Result doWork() {
 
 
     private long getTimeStamp() {
```

Comment:
Misschien even een docstring toevoegen die meteen duidelijk maakt om welke tijdzone het gaat? (ik gok gewoon UTC?)

Reply:
The java.util.Date.getTime() method returns how many milliseconds have passed since January 1, 1970, 00:00:00 GMT

Reply:
jep =D